### PR TITLE
Add "wait_for_routes" helper script for CronJobs.

### PR DESCRIPTION
**Pull request by bpblanken**

### CODE REVIEW

File: deploy/docker/seqr/wait_for_routes [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+###
+# Waits for network endpoints.  Intended useage is within Kubernetes CronJobs to wait for sidecar availability.
```
Comment (sjahl) [inline deploy/docker/seqr/wait_for_routes:4]: don't want to be that guy, but `s/useage/usage` 😄

**Review by sjahl — APPROVED** [review]
the Bash all looks good to me here, just the one spelling mistake

File: deploy/docker/seqr/wait_for_routes (hunk 1 above)
Comment (bpblanken) [reply deploy/docker/seqr/wait_for_routes:4]: ❤️ that the typo is in the comment and not the bash

**Comment by hanars** [comment]
all seqr PRs should be against dev, not master, unless there is an overwhelmingly compelling reason why we want to release it ahead of everything else on dev. If that is the case, then you need to fill out the [release checklist](https://github.com/broadinstitute/seqr-private/blob/master/release_pr_checklist.md) and make sure to a do a deployment as soon as it is merged

**Comment by bpblanken** [comment]
@hanars I've pointed at `dev`.

**Review by hanars — APPROVED** [review]

**Pull request merged**